xcm-simulator/src/storagehub/configs/mod.rs

```diff
@@ -707,13 +692,11 @@ impl Convert<<Runtime as frame_system::Config>::Hash, ThresholdType>
         // Get the hash as bytes
         let hash_bytes = hash.as_ref();
 
-        // Get the 16 least significant bytes of the hash and interpret them as a u128
-        let truncated_hash_bytes: [u8; 16] =
-            hash_bytes[16..].try_into().expect("Hash is 32 bytes; qed");
-        let hash_as_u128 = u128::from_be_bytes(truncated_hash_bytes);
+        // Get the 4 least significant bytes of the hash and interpret them as an u32
+        let truncated_hash_bytes: [u8; 4] =
+            hash_bytes[28..].try_into().expect("Hash is 32 bytes; qed");
```

Comment:
For all runtimes including mocks, I set the threshold type to `u32`. Therefore I take now 4 bytes instead of 16 (threshold type was `u128`) from the hash.

Reply:
Right, so shouldn't it be `4`? I believe `hash_bytes` is a stream of bytes, not bits. So you don't need the first 28 bytes, just the first 28 bits (4 bytes)

Reply:
Yes, it is an array of bytes. We are grabbing the 4 least significant bytes (equaling 32 bits), fitting into the 32 bit unsigned integer threshold type.

Reply:
Right, so shouldn't this be 4 instead of 28 then?